Lab6.ipynb

```diff
@@ -0,0 +1,692 @@
+      "The upper limit 5 sigma theshold is 154.0 and the lower limit 5 sigma threshold is 54.0 for the LHC background.\n"
+     ]
+    }
+   ],
+   "source": [
+    "#using stats.norm.sf to get the probability of measuring 5sigma or greater of a Gaussian\n",
+    "prob_5sigma = stats.norm.sf(5)\n",
+    "\n",
+    "#using stats.poisson.isf and stats.poisson.ppf to get the upper and lower limits, respectively of the 5sigma threshold on the background distribution\n",
+    "upper_threshold = stats.poisson.isf(prob_5sigma, 100)\n",
+    "lower_threshold = stats.poisson.ppf(prob_5sigma, 100)\n",
+    "print('The upper limit 5 sigma theshold is {} and the lower limit 5 sigma threshold is {} for the LHC background.'.format(upper_threshold, lower_threshold))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here it can be seen that the 5$\\sigma$ values are not symmetric about the mean of 100, which is consistent with the characteristics of the Poisson background distribution. They are relatively close to being symmetric, since N=100,000 is a large number of simulations and thus the distribution is approaching that of a Gaussian."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here I am following the same procedure as in Problem 1, except I am finding the threshold value for 8$\\sigma$ instead of 5$\\sigma$, and I am only concerned about the upper bound. I assume that potential signals happen above the 100-mean count background. I am finding a signal that is strong enough that it can be discriminated from the background vary easily, so that I can see how even an obvious detection is affected by the background."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The 8 sigma value of this distribution is 190.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "#finding the 8 sigma threshold\n",
+    "prob_8sigma = stats.norm.sf(8)\n",
+    "threshold = stats.poisson.isf(prob_8sigma, 100)\n",
+    "print('The 8 sigma value of this distribution is {}'.format(threshold))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to inject a signal of 190 into the background noise, I am adding the value 190 to 100,000 randomly selected values from the simulated background distribution. This will simulate many possible measurements of event with the same true signal, but are smeared by the background noise. Even if the signal source is giving the exact same number of counts each time, because of the background, there will be a number of possible observed signals that may or may not be the true signal value. I have plotted the histogram of the possible observed signals below, on regular and log scale plots."
```

Comment:
I don't understand why take normally distributed indices from a Poisson distributed background. You could just add 190 to the background dist generated in problem 1 to get a more accurate effect.

Reply:
Meant to say randomly, not normally.